src/components/notifications/SuccessModal.tsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react'
+import { BaseOpacity, Box, mkUseStyles, Text } from 'utils/theme'
+import { Image } from 'react-native'
+
+type ModalPropsType = {
+  title: string
+  onPressText?: string
+  onPress?: F0
+}
+
+export const SuccessModal = ({ title, onPressText, onPress }: ModalPropsType) => {
+  const styles = useStyles()
+
+  return (
+    <Box
+      padding="m"
+      backgroundColor="successToastBg"
+      minHeight={64}
+      alignItems="center"
+      borderWidth={1}
+      borderColor="approvedGreen"
+      borderRadius="l1min"
+      flexDirection="row">
+      <Image source={require('assets/icons/success-icon.png')} style={styles.image} />
```

Comment:
`const successIcon = require('assets/icons/success-icon.png')`  
You can move the icon out of React scope to avoid loading it on each rerender.

`<Image source={successIcon} style={styles.image} />`